app/models/concerns/node_shared.rb

```diff
@@ -53,6 +53,23 @@ def self.graph_grid(body, _page = 1)
     end
   end
 
+  def self.simple_data_grapher(body, _page = 1)
```

Comment:
I think we can extract the similar parts of this method and the previous method and reuse...I will try to refactor a little and let you know.

Reply:
We can try that, but maybe it would be best to keep it separate, you know so that it's easier to change in the future?